src/Sulu/Bundle/SecurityBundle/Tests/Functional/Command/CreateUserCommandTest.php

```diff
@@ -18,10 +18,8 @@
 
 class CreateUserCommandTest extends SuluTestCase
 {
-    /**
-     * @var CommandTester
```

Comment:
Please don't remove the PHPDocs, they are very useful if you use an IDE. And can you also add one to the other member variable?